src/__tests__/index.tsx

```diff
@@ -89,24 +85,30 @@ export default ({children}) => <div className="sub-dir">{children}</div>
         </p>
       </header>
       <main>
-        <h1>
-          This is the title
-        </h1>
-        <p>
-          Here's a 
-          <strong>
-            neat
-          </strong>
-           demo:
-        </p>
-        <div>
-          !!Neat demo!
-          <div
-            class="sub-dir"
-          >
-            Sub dir!
+        <wrapper
+          mdxtype="MDXLayout"
+        >
```

Comment:
Yeah this is what I was talking about in the PR, they appeared without me modifying the mdx compiler so not sure. Reading the other comments here it seems like adding XDM is worth doing now so I'll perserve with that and see what happens.

Reply:
Back on the xdm idea, would it be possible to configure jest to compile files that are in `node_modules` rather than trying to make native modules work?